docs/ipv6.md

```diff
@@ -0,0 +1,170 @@
+## Setting up an IPv6 Lab using kube-centos-ansible
+
+This document covers a simple setup of using kube-centos-ansible for spinning up an IPv6 laboratory based in part by this [customized release of Kubernetes](https://github.com/leblancd/kubernetes/releases/tag/v1.9.0-alpha.1.ipv6.1b) created by [leblancd](https://github.com/leblancd) and automated herein by [fepan](https://github.com/fepan).
+
+In part this document assumes some other familiarity with these Ansible playbooks. In theory you should be able to spin up just an IPv6 lab without otherwise having experience with this playbook. However, some of the terminology may be glossed over. We often use the term "virthost" -- by which we mean a host that runs virtual machines, this technique is used often by the developers of this project in order to quickly iterate on these playbooks. These instructions assume using a virthost, however, it's likely you could complete a deployment on baremetal, or on an IaaS.
+
+## Limitations
+
+Currently, this setup makes it possible to `ping6` one pod from another. We look forward to using this laboratory to explore the other possibilities and scenarios herein, however this pod-to-pod `ping6` is the baseline functionality from which to start.
+
+## Requirements
+
+This also assumes that you have:
+
+* A machine for running Ansible (like your workstation) and [have Ansible installed](http://docs.ansible.com/ansible/latest/intro_installation.html).
+* Ansible 2.4 or later (necessary to support `get_url` with IPv6 enabled machines)
+* A host capable of running virtual machines, and is running CentOS 7.
+* A clone of this repository.
+
+This scenario disables the "bridged networking" feature we often use and instead uses NAT'ed libvirt virtual machines. 
+
+You may have to disable GRO ([generic receive offload](https://en.wikipedia.org/wiki/Large_receive_offload)) for the NICs on the virtualization host (if you're using one).
+
+An example of doing so is:
+
+```
+ethtool -K em3 gro off
+```
+
+## Inventory and variable setup
+
+Firstly, let's look at an inventory and variable overrides to use.
+
+Here's the initially used inventory, which only really cares about the `virthost`. These examples place the inventory file @ `inventory/your.virthost.inventory`. You'll need to modify the location of the host to match your environment.
+
+```
+the_virthost ansible_host=192.168.1.119 ansible_ssh_user=root
+
+[virthost]
+the_virthost
+```
+
+And the overrides which are based on the examples @ `./inventory/examples/virthost/virthost-ipv6.inventory.yml`. In these examples this set of extra variables is located @ `./inventory/extravars.yml`:
```

Comment:
extravars? huh... that's new :)  I'd have expected this in a group_vars/virthost.yml